tsdb/db.go

```diff
@@ -66,20 +77,17 @@ type Options struct {
 	// WALSegmentSize = 0, segment size is default size.
 	// WALSegmentSize > 0, segment size is WALSegmentSize.
 	// WALSegmentSize < 0, wal is disabled.
-	WALSegmentSize int
+	WALSegmentSize units.Base2Bytes
 
 	// Duration of persisted data to keep.
-	RetentionDuration uint64
+	RetentionDuration model.Duration
 
 	// Maximum number of bytes in blocks to be retained.
 	// 0 or less means disabled.
 	// NOTE: For proper storage calculations need to consider
 	// the size of the WAL folder which is not added when calculating
 	// the current size of the database.
-	MaxBytes int64
-
-	// The sizes of the Blocks.
-	BlockRanges []int64
```

Comment:
It's not clear to me what was this replaced with.

Reply:
It is not controlled by Min and Max Block sizes. It can be overrided by tests as well.